### PR TITLE
fix: Fixed problems with singlescan and decreased .sql files count

### PR DESCRIPTION
**Pull request by v-kerimov**
# Pull Request

## Description

Fixed problems with singlescan scenario database initialization and got rid of redundant .sql files.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Code cleanup/refactoring
- [ ] Documentation update
- [ ] This change requires a documentation update
- [ ] CI system update
- [ ] Test Coverage update

## Testing

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

**Test Configuration**:
* Java: v11
* LPVS Release: v1.x.x

# Checklist:

- [ ] My code follows the style guidelines of this project
- [x] My code meets the required code coverage for lines (90% and above)
- [ ] My code meets the required code coverage for branches (80% and above)
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/main/resources/database_dump.sql [hunk 1]
```diff
@@ -1,190 +1,88 @@
...
+  id bigint PRIMARY KEY NOT NULL AUTO_INCREMENT,
+  email varchar(255) NOT NULL,
+  name varchar(255) NOT NULL,
+  nickname varchar(255) DEFAULT NULL,
+  provider varchar(255) NOT NULL,
+  organization varchar(255) DEFAULT NULL,
+  UNIQUE (email,provider)
+);
+
+INSERT INTO licenses (id, license_alternative_names, license_name, license_spdx, license_usage) VALUES
```
Comment (tiokim) [inline src/main/resources/database_dump.sql:81]: What about excluding `id` and `license_alternative_names`?

```suggestion
INSERT INTO licenses (license_name, license_spdx, license_usage) VALUES
```

**Review by tiokim — COMMENTED** [review]
PTAL

File: src/main/resources/application-singlescan.properties [hunk 2]
```diff
@@ -1,7 +1,22 @@
...
+spring.datasource.url=jdbc:h2:mem:singlescan;DATABASE_TO_UPPER=false;MODE=MySQL;NON_KEYWORDS=USER
+spring.datasource.username=sa
 spring.datasource.password=
+
+spring.jpa.properties.hibernate.format_sql=false
+spring.jpa.properties.hibernate.use_sql_comments=false
+spring.jpa.properties.hibernate.show_sql=false
+
+spring.jpa.hibernate.ddl-auto=create-drop
+spring.jpa.defer-datasource-initialization=true
```
[the remark this answers is not in the document]
Comment (o-kopysov) [reply src/main/resources/application-singlescan.properties:22]: @o-konoval  Please check whether we need to modify YML file or not.

[the remark this answers is not in the document]
Comment (o-konoval) [reply src/main/resources/application-singlescan.properties:22]: I checked this PR- using docker scenario-dump seems imported normally without changing
 [docker-compose.yml](https://github.com/Samsung/LPVS/blob/main/docker-compose.yml#L12)
Log:
lpvs-lpvs-1     | 2023-12-06 12:40:24.192  INFO 1 --- [           main] j.LocalContainerEntityManagerFactoryBean : Initialized JPA EntityManagerFactory for persistence unit 'default'
lpvs-lpvs-1     | 2023-12-06 12:40:25.574  INFO 1 --- [           main] com.lpvs.service.LPVSLicenseService      : LICENSES: loaded 7 licenses from DB.
lpvs-lpvs-1     | 2023-12-06 12:40:25.580  INFO 1 --- [           main] com.lpvs.service.LPVSLicenseService      : LICENSE CONFLICTS: loaded 0 license conflicts from DB.
lpvs-lpvs-1     | 2023-12-06 12:40:25.584  INFO 1 --- [           main] com.lpvs.service.LPVSDetectService       : License detection scanner: scanoss
...
...
...

lpvs-lpvs-1     | 2023-12-06 13:48:22.959  INFO 1 --- [           main] c.l.service.LPVSQueueProcessorService    : PROCESS Webhook id = 1
lpvs-lpvs-1     | 2023-12-06 13:48:22.964  INFO 1 --- [  LPVS-ASYNC::1] com.lpvs.service.LPVSQueueService        : GitHub Webhook processing...
lpvs-lpvs-1     | 2023-12-06 13:48:22.965  INFO 1 --- [  LPVS-ASYNC::1] com.lpvs.service.LPVSQueueService        : com.lpvs.entity.LPVSQueue@b93ae032


For me seems these parameters do not take influence on import of dump for docker scenario.

File: src/main/resources/database_dump.sql [hunk 3]
```diff
@@ -1,190 +1,101 @@
...
+  nickname varchar(255) DEFAULT NULL,
+  provider varchar(255) NOT NULL,
+  organization varchar(255) DEFAULT NULL,
+  UNIQUE (email,provider)
+);
+
+INSERT INTO licenses (license_name, license_spdx, license_usage) VALUES
+('GNU General Public License v3.0 only','GPL-3.0-only','PROHIBITED'),
+('OpenSSL License','OpenSSL','PERMITTED'),
+('GNU Lesser General Public License v2.0 or later','GPL-2.0-or-later','RESTRICTED'),
```
Comment (tiokim) [inline src/main/resources/database_dump.sql:97]:
```suggestion
('GNU Lesser General Public License v2.0 or later','LGPL-2.0-or-later','RESTRICTED'),
```

**Review by o-kopysov — APPROVED** [review]
I think that empty lines could be added at the end of each file. Other looks fine for me.

**Review by userai-dev — APPROVED** [review]
Good job! Approved!

**Review by o-konoval — APPROVED** [review]
LGTM

**Review by m-rudyk — APPROVED** [review]
I have tested this changes and approve working.

**Pull request merged**